pkg/sql/opt/exec/execbuilder/testdata/inverted_index

```diff
@@ -3,12 +3,49 @@
 statement ok
 CREATE TABLE d (
   a INT PRIMARY KEY,
-  b JSONB
+  b JSONB,
+  FAMILY (a,b)
 )
 
 statement ok
 CREATE INVERTED INDEX foo_inv ON d(b)
 
+query T kvtrace
+INSERT INTO d VALUES(0, '{"a": "b"}')
+----
+CPut /Table/53/1/0/0 -> /TUPLE/
```

Comment:
This seems strange -- where b in the value?

Reply:
you mean the /TUPLE/ thing? I think we don't have something defined for arrays or json - it's like this with or without my patch.

Reply:
We didn't implement PrettyPrintValueEncoded for these types. I'll send a separate PR for that.